checkov/terraform/checks/resource/gcp/DataFusionPrivateInstance.py

```diff
@@ -0,0 +1,18 @@
+from checkov.terraform.checks.resource.base_resource_value_check import BaseResourceValueCheck
+from checkov.common.models.enums import CheckCategories
+from checkov.common.models.consts import ANY_VALUE
+
+
+class DataFusionPrivateInstance(BaseResourceValueCheck):
+    def __init__(self):
+        name = "Ensure Data fusion instances are private"
+        id = "CKV_GCP_87"
+        supported_resources = ['google_data_fusion_instance']
+        categories = [CheckCategories.GENERAL_SECURITY]
+        super().__init__(name=name, id=id, categories=categories, supported_resources=supported_resources)
+
+    def get_inspected_key(self):
+        return 'private_instance'
```

Comment:
do you know if the default is `true` or `false`?

Reply:
Sounds like public by default to me? `[private_instance](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/data_fusion_instance#private_instance) - (Optional) Specifies whether the Data Fusion instance should be private. If set to true, all Data Fusion nodes will have private IP addresses and will not be able to access the public internet. `

Reply:
<img width="941" alt="image" src="https://user-images.githubusercontent.com/1456880/153918390-bb08bcf9-c3aa-4961-8444-2b35360a4479.png">

Reply:
opt in for private :(

Reply:
<img width="941" alt="image" src="https://user-images.githubusercontent.com/1456880/153918538-67b9854a-e329-4d00-8aa0-13afc32c4317.png">

Reply:
and so it is. tsk tsk google.